src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    add(a, b) {
+      return (a + b);
```

Comment:
everything is good, just small advice - it can be simplified by omitting the parentheses